make-base/build.mk

```diff
@@ -0,0 +1,19 @@
+.PHONY: always
```

Comment:
This is the main WIP -- the idea is to template out the global namespaced targets that should make declaring and using inter project image building and dependencies more sane.  You'll be able to declare that vulcan's image depends on archimedes image without a direct `make -C archimedes` invocation and a bunch of weird state files.